mission_yaml/SEA70_M14.yml

```diff
@@ -0,0 +1,601 @@
+metadata:
+  acknowledgement: This study used data collected and made freely available by Voice of the Ocean Foundation
+                   (https://voiceoftheocean.org) accessed from https://erddap.observations.voiceoftheocean.org/erddap/index.html
+  institution: Voice of the Ocean Foundation
+  license: Creative Commons Attribution 4.0 (https://creativecommons.org/licenses/by/4.0/)
+    This study used data collected and made freely available by Voice of the Ocean Foundation
+    (https://voiceoftheocean.org) accessed from https://erddap.observations.voiceoftheocean.org/erddap/index.html
+  format_version: IOOS_Glider_NetCDF_v2.0.nc
+  glider_model: SeaExplorer
+  glider_instrument_name: seaexplorer
+  keywords: "AUVS, Autonomous Underwater Vehicles, Oceans, Ocean Pressure,
+             Water Pressure, Ocean Temperature, Water Temperature,
+             Salinity/Density, Conductivity, Density, Salinity"
+  keywords_vocabulary: GCMD Science Keywords
+  metadata_link: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html
+  Metadata_Conventions: CF-1.6, Unidata Dataset Discovery v1.0
+  naming_authority: Voice of the Ocean Foundation
+  platform:    "SeaExplorer Glider"
+  processing_level: "Data provided as is with no expressed or implied
+                     assurance of quality assurance or quality control. L0"
+  publisher_email: callum.rollo@voiceoftheocean.org
+  publisher_name:  Callum Rollo
+  publisher_url:   https://voiceoftheocean.org
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  sea_name: Baltic
+  transmission_system: IRIDIUM
+  glider_name: Scratch
+  glider_serial: "70"
+  wmo_id: "Nan"
+  comment: 'deployment and recovery in Bornholm. Metadata missing and methane sensor data. No salinity data-just conductivity '
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
+  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician
+  creator_email: callum.rollo@voiceoftheocean.org
+  creator_name: Callum Rollo
+  creator_url: https://observations.voiceoftheocean.org
+  deployment_id: "14"
+  deployment_name: NS_Bornholm
+  # date format 'yyyy-mm-dd'
+  deployment_start: "2022-10-20"
+  deployment_end: "2022-11-20"
+  project: NS_Bornholm
+  project_url: https://voiceoftheocean.org/
+  summary: ' Part of NS_Bornholm continuous monitoring'
+
+
+glider_devices:
+  ctd:
+    make: Seabird
+    model: GPCTD
+    serial: 'Nan'
+    long_name: Seabird SlocumCTD
```

Comment:
Get the serial numbers that you uploaded for SEA070_M13. I recommend starting this PR again  after updating your fork so you work from the most recent version of SEA070_M13 with the improvements we've to it